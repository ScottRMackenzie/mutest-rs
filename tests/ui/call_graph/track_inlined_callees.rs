//@ print-call-graph
//@ print-targets
//@ stdout
//@ stderr: empty
//@ rustc-flags: -C opt-level=3

fn called_from_inlined_fn() {}

#[inline]
fn inlined_fn() {
    called_from_inlined_fn();
}

#[test]
fn test() {
    inlined_fn();
}
