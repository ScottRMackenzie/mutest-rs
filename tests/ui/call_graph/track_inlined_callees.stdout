
@@@ call graph @@@

entry points:

test test
  -> inlined_fn at tests/ui/call_graph/track_inlined_callees.rs:10:1: 10:16 (#0)

nested calls at distance 1:

inlined_fn at tests/ui/call_graph/track_inlined_callees.rs:10:1: 10:16 (#0)
  -> called_from_inlined_fn at tests/ui/call_graph/track_inlined_callees.rs:7:1: 7:28 (#0)

@@@ targets @@@

tests -(1)-> called_from_inlined_fn at tests/ui/call_graph/track_inlined_callees.rs:7:1: 7:28 (#0)
  (1) test

tests -(0)-> inlined_fn at tests/ui/call_graph/track_inlined_callees.rs:10:1: 10:16 (#0)
  (0) test

targets: 2 total; 2 safe; 0 unsafe (0 tainted)
